src/js/ui/editor/menu.js

```diff
@@ -80,6 +80,24 @@ export function renderSelectColorPosition(ctrl) {
         )
       ])
     ]),
+    m('div.select_input', [
+      m('label', {
+        'for': 'select_editor_positions'
+      }, 'Endgames'),
+      m('select.positions', {
+        id: 'select_editor_positions',
+        onchange: function(e) {
+          ctrl.loadNewFen(e.target.value);
+        }
+      }, [
+        optgroup('Set the board', [
```

Comment:
Ok it working like before. I think that line ' chromium-browser --user-data-dir=$HOME/.chromium_dev_dir --disable-web-security
 ' should remains in the readme shouldn't it ?